config/csm/test_rest_capacity.yaml

```diff
@@ -137,3 +134,25 @@ test_41156:
   min_users: 5
   max_users: 100
   available_size: 1024
+test_41964:
```

Comment:
I see lot of config values are common, can we combine them instead in common params unless they are absolutely test specific? @aditipande96  @kachhwahadivya

Reply:
We can could do for some but then if in future they change for one test and not for other we are in trouble. So common parameter we put in constants.py or csm_config.yaml and here we have different ones.